wordpress-mobile/WordPress-iOS
# Issue/media coordinator on legacy

This PR refactors the legacy editor to use the new MediaProgressCoordinator created for Aztec.

This avoid a lot of code duplication in the editor for keeping track of progress reporting of files.

To test:
 - Start the iOS
 - Activate the legacy editor
 - Start to write a new post
 - Add new media (image, videos) to the post and see if they are inserted and progress reporting is done correctly.
 - Check error scenarios when uploads fail.


Needs review: @astralbodies
